string target_name
geometry_msgs/Pose[] poses_top
geometry_msgs/Pose[] poses_center
geometry_msgs/Pose[] poses_bottom

#For each time step